### ISSUE TITLE
Swap from `sassc-rails` to `dartsass-rails`? Or Deprecate this gem entirely? 

### ISSUE THREAD

**Issue by jmarchello** [rails/sass-rails#451]
Seeing as how `sassc-rails` is unmaintained and [libsass is deprecated](https://sass-lang.com/blog/libsass-is-deprecated/), Should we update this gem to wrap `dartsass-rails` instead? If not, should this gem be deprecated?

**Comment by rgaufman** [comment]
Yes, it seem this gem is deprecated and should not be used anymore. Unfortunately so many other gems depend on this gem, e.g. font-awesome-sass, font-awesome-rails, bootstrap, etc.
